exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -237,12 +238,12 @@ func makeAws(attributes map[string]pcommon.Value, resource pcommon.Resource, log
 		cwl = getLogGroupMetadata(configSlice, false)
 	}
 
-	if sdkName != "" && sdkLanguage != "" {
+	sdkLanguageSuffix := fmt.Sprintf(" for %s", sdkLanguage)
+	sdk = sdkName
+	if sdkName != "" && sdkLanguage != "" && !strings.HasSuffix(strings.ToLower(sdkName), strings.ToLower(sdkLanguageSuffix)) {
```

Comment:
Seems a shame we have to use `ToLower()`. Hopefully performance impact is negligible.

Reply:
We just have no guarantee that the cases will be the same, so better to normalize both before doing the check.